models/boost/boost_claimed.sql

```diff
@@ -44,7 +44,10 @@ quest_claims_enriched as (
         coalesce(b.reward_token_address, c.reward_token_address) reward_token_address,
         claim_tx_hash,
         c.block_time,
-        coalesce(claim_fee_eth, 0.000075) claim_fee_eth,
+        case 
+            when block_time >= timestamp '2024-06-06 16:00:00' then 0.0
```

Comment:
We may introduce these timestamps as jinja variables.

Reply:
@Hosuke do you mind pushing a commit to show how it's done? then @pntemi can leverage these things moving forward 🤝

Reply:
Just added.

Reply:
i think the variables need single quotes around them, went ahead and pushed. let's see CI run now.

Reply:
looks to work now

Reply:
Forgot to push the local commits.